pkg/tss/tpm_linux.go

```diff
@@ -25,7 +25,7 @@ func probeSystemTPMs() ([]ProbedTPM, error) {
 	var tpms []ProbedTPM
 
 	tpmDevs, err := ioutil.ReadDir(tpmRoot)
-	if err != nil && !os.IsNotExist(err) {
+	if err != nil {
 		return nil, err
 	}
 	if err == nil {
```

Comment:
this is still an issue, the err == nil check

and i think there was a point to doing this before, but it wasn't very clear why.